### PR TITLE
Dwarf Power

### PR DESCRIPTION
**Pull request by B0ZIA**
I have changed the way HP and damage are calculated in the game so that it is easier to determine how much godda the player will get for defeating them and to calculate item parameters

### CODE REVIEW

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd [hunk 1]
```diff
@@ -2,20 +2,18 @@ extends SpawnerSystem
...
-export(float) var boss_damage : float = 2.0
 export(bool) var spawn : bool = true
 
 var Dwarf = load("res://Objects/Dwarves/Dwarf/Dwarf.tscn")
 var Boss = load("res://Objects/Dwarves/Boss/Boss.tscn")
 
-# dwarf data on level 1
 onready var default_dwarf_hp = dwarf_max_hp
 onready var default_dwarf_damage = dwarf_damage
+onready var WPHK = 0.0305
```
Comment (Ajver) [inline Objects/Spawners/DwarvesManager/DwarvesManager.gd]: This name means nothing. I recommend to write something like this: `hp_increase_ratio` and: `damage_increase_ratio`.

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd [hunk 2]
```diff
@@ -2,20 +2,18 @@ extends SpawnerSystem
...
+onready var WPDK = 0.01894
+onready var BOSS_DAMAGE_MULTIPLAY = 1.5
```
Comment (Ajver) [inline Objects/Spawners/DwarvesManager/DwarvesManager.gd]: Little typo: should be `_MULTIPLY` (without 'a')

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd [hunk 3]
```diff
@@ -2,20 +2,18 @@ extends SpawnerSystem
...
+onready var BOSS_HP_MULTIPLAY = 1.75
```
Comment (Ajver) [inline Objects/Spawners/DwarvesManager/DwarvesManager.gd]: Little typo: should be `_MULTIPLY` (without 'a')

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd (hunk 1 above)
Comment (B0ZIA) [reply Objects/Spawners/DwarvesManager/DwarvesManager.gd]: Yes, you are right

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd (hunk 3 above)
Comment (B0ZIA) [reply Objects/Spawners/DwarvesManager/DwarvesManager.gd]: ok

File: Objects/Spawners/DwarvesManager/DwarvesManager.gd (hunk 2 above)
Comment (B0ZIA) [reply Objects/Spawners/DwarvesManager/DwarvesManager.gd]: ok

**Pull request merged**